scripts/globals/status.lua

```diff
@@ -78,75 +78,80 @@ dsp.status =
-SUBEFFECT_DETONATION    = 13;
-SUBEFFECT_IMPACTION     = 14;
+dsp.subEffects =
+{
+    -- ATTACKS
+    FIRE_DAMAGE         = 1,   -- 110000        3
+    ICE_DAMAGE          = 2,   -- 1-01000       5
+    WIND_DAMAGE         = 3,   -- 111000        7
+    CHOKE               = 3,   -- Shares subeffect
+    EARTH_DAMAGE        = 4,   -- 1-00100       9
+    LIGHTNING_DAMAGE    = 5,   -- 110100       11
+    WATER_DAMAGE        = 6,   -- 1-01100      13
+    LIGHT_DAMAGE        = 7,   -- 111100       15
+    DARKNESS_DAMAGE     = 8,   -- 1-00010      17
+    DISPEL              = 8,   -- Verified with video of Lockheart Greatsword proc.
+    SLEEP               = 9,   -- 110010       19
+    POISON              = 10,  -- 1-01010      21
+    PARALYSIS           = 11,
+    AMNESIA             = 11,  -- Verified uses same animation as para
+    BLIND               = 12,  -- 1-00110      25
+    SILENCE             = 13,
+    PETRIFY             = 14,
+    PLAGUE              = 15,
+    STUN                = 16,
+    CURSE               = 17,
+    DEFENSE_DOWN        = 18,  -- 1-01001      37
+    EVASION_DOWN        = 18,  -- Same subeffect as DEFENSE_DOWN
+    ATTACK_DOWN         = 18,  -- Same subeffect as DEFENSE_DOWN
+    DEATH               = 19,
+    SHIELD              = 20,
+    HP_DRAIN            = 21,  -- 1-10101      43
+    MP_DRAIN            = 22,  -- This is correct animation
+    TP_DRAIN            = 22,  -- Verified this should look exactly like Aspir Samba.
+    HASTE               = 23,
+    -- There are no additional attack effect animations beyond 23. Some effects share subeffect/animations.
+
+    -- SPIKES
+    BLAZE_SPIKES        = 1,   -- 01-1000       6
+    ICE_SPIKES          = 2,   -- 01-0100      10
+    DREAD_SPIKES        = 3,   -- 01-1100      14
+    CURSE_SPIKES        = 4,   -- 01-0010      18
+    SHOCK_SPIKES        = 5,   -- 01-1010      22
+    REPRISAL            = 6,   -- 01-0110      26
+    GLINT_SPIKES        = 6,   --
+    GALE_SPIKES         = 7,   -- Used by enchantment "Cool Breeze" http://www.ffxiah.com/item/22018/
+    CLOD_SPIKES         = 8,   --
+    DELUGE_SPIKES       = 9,   --
+    DEATH_SPIKES        = 10,  -- yes really: http://www.ffxiah.com/item/26944/
+    COUNTER             = 63,
+    -- There are no spikes effect animations beyond 63. Some effects share subeffect/animations.
+    -- "Damage Spikes" use the Blaze Spikes animation even though they are different status.
+
+    -- SKILLCHAINS
+    NONE                = 0,
+    LIGHT               = 1,
+    DARKNESS            = 2,
+    GRAVITATION         = 3,
+    FRAGMENTATION       = 4,
+    DISTORTION          = 5,
+    FUSION              = 6,
+    COMPRESSION         = 7,
+    LIQUEFACATION       = 8,
+    INDURATION          = 9,
+    REVERBERATION       = 10,
+    TRANSFIXION         = 11,
+    SCISSION            = 12,
+    DETONATION          = 13,
+    IMPACTION           = 14,
+};
+dsp.sub = dsp.subEffects;
```

Comment:
```lua
dsp.subEffect
```
is superior to both sub and subEffects in my opinion, because you already depluralized (is that a word? w/e we're gonna pretend it is) the others and sub is treated as a keyword in several syntax parsers:
![image](https://user-images.githubusercontent.com/6871475/39401850-c557e960-4b1d-11e8-9226-8bb97cc3eca6.png)

Which may possibly cause future confusion (or it might not, but just my 2 cents here)

Reply:
agreed. fixed.